### PR TITLE
Change time variable type to float

### PR DESCRIPTION
**Pull request by ghsdh3409**
I changed all of variable type related to time to float.

### CODE REVIEW

File: android/src/main/java/com/inprogress/reactnativeyoutube/YouTubeView.java [hunk 1]
```diff
@@ -123,7 +123,7 @@ public void playerViewDidBecomeReady() {
     public void didChangeToSeeking(int milliSeconds) {
         WritableMap event = Arguments.createMap();
         event.putString("state", "seeking");
-        event.putInt("currentTime", milliSeconds / 1000);
+        event.putDouble("currentTime", milliSeconds / 1000.0);
```
Comment (ghsdh3409) [inline android/src/main/java/com/inprogress/reactnativeyoutube/YouTubeView.java:126]: No exists `putFloat`

**Comment by ghsdh3409** [comment]
@ohtangza I will merge this pull request without your review to test this! (Test is little inconvenient since podspec...)

File: android/src/main/java/com/inprogress/reactnativeyoutube/YouTubeView.java (hunk 1 above)
Comment (ohtangza) [reply android/src/main/java/com/inprogress/reactnativeyoutube/YouTubeView.java:126]: Checked. Thanks! Little interesting that the original author used sometime milliSecs and sometimes secs.

**Review by ohtangza — COMMENTED** [review]
I checked all the changes!

**Pull request merged**